usaspending_api/common/helpers.py

```diff
@@ -48,6 +48,13 @@ def generate_fiscal_month(date):
     return date.month + 3
 
 
+def get_pagination(results, limit, page):
```

Comment:
Pagination was the largest performance hit on the aggregate endpoints. Is there any benchmarking to the performance with and without the pagination?